### ISSUE TITLE
Include date information in loading completion message

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse-catalog#468]
## Problem

Our load completion reporting Slack message currently does not include any date information. For instances where `catchup=True` and we need to re-run multiple days, it can be hard to distinguish which runs came from which days.

## Description

It would be nice to have a date range specified for each DAG. E.g. `@daily` would have just the day it ran on, say `2022-04-08`. `@weekly` could be `2022-04-01 -> 2022-04-08`. DAGs that consume the entire API could say `date: all` or something.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by zackkrida** [comment]
I definitely support adding this date information—could we _also_ add a permalink to the Airflow UI? In the future, it would be so cool to be able to give [public access](https://airflow.apache.org/docs/apache-airflow/stable/security/access-control.html) to things like DAG runs.

**Comment by AetherUnbound** [comment]
I hadn't even considered that, every Airflow I've worked with has been behind a firewall :open_mouth: I love it! And a permalink would also be easy to add.

**Comment by AetherUnbound** [comment]
This should be completed before #380

**Comment by AetherUnbound** [comment]
It would also be great to note which ones are `dated` and which ones grab all of the API data

**Comment by zackkrida** [comment]
> It would also be great to note which ones are dated and which ones grab all of the API data

Would `dated` be a good tag in Airflow for these provider DAGs?

**Comment by AetherUnbound** [comment]
> Would `dated` be a good tag in Airflow for these provider DAGs?

It would be, fantastic idea!
